hedera-node/hedera-app/src/main/java/com/hedera/node/app/blocks/impl/streaming/FileBlockItemWriter.java

```diff
@@ -237,6 +239,16 @@ public static List<OnDiskPendingBlock> loadContiguousPendingBlocks(
                         Arrays.toString(Arrays.copyOfRange(proofJsons.toArray(), i + 1, proofJsons.size())));
                 break;
             }
+            // Verify old proofs without block timestamp or sibling hashes are skipped
+            if (pendingProof.blockTimestamp() == null
+                    || Objects.equals(pendingProof.blockTimestamp(), Timestamp.DEFAULT)
+                    || pendingProof.siblingHashesFromPrevBlockRoot().size() != 4) {
+                logger.warn(
+                        "Pending proof metadata from {} is missing required fields (not considering remaining - {})",
```

Comment:
Why do we need to log warning here ?

Reply:
Logged this since the code doesn't try to reload any more blocks. And also because it logs a similar message [just above](url).